### PR TITLE
Describe default read_policies in docstring

### PR DESCRIPTION
**Pull request by dcslagel**
#### Description:

This change it to address the documentation updates for #332 "Additional curves 'UNKNOWN:1' added and phantom columns made".

##### Work so far..
- Add description of the defaults.READ_POLICIES to
  reader.get_substitutions()
- remove run-on(NaN.) from defaults.READ_POLICIES['defaults'].
  It's functionality is included in run-on(.).
- Add Arguments information to reader.find_sections_in_file() docstring
- Expand the discussion in data_section.rst on handling run-on errors

--

Let me know if this change could be accepted (or rejected) or
needs some additional changes to be approved and merged.

Thank you,
DC

### CODE REVIEW

File: docs/source/data-section.rst [hunk 1]
```diff
@@ -94,7 +94,7 @@ This is represented in the following way:
 Note that the actual mnemonic is not present, to avoid ambiguity about
 which curve would be expected to be returned:
 
-.. code-block:: python
+.. code-block::
```
Comment (dcslagel) [inline docs/source/data-section.rst:97]: sphinx was complaining about this not being python code so I remove 'python'.

**Comment by dcslagel** [comment]
Hi @kinverarity1, 

Could you give this a review?  It tries to cover the topics brought up in  #332 "Additional curves 'UNKNOWN:1' added and phantom columns made".   Let me know if any area needs to be improved or added to.

Thanks,
DC

**Review by kinverarity1 — APPROVED** [review]
Thank you for doing this. It's excellent and very helpful!

File: docs/source/data-section.rst (hunk 1 above)
Comment (dcslagel) [reply docs/source/data-section.rst]: Note: I changed this in the back to `.. code-block:: python` and added a second block:  `.. code-block:: console` for the output.  This enables the python code to be highlight but separates it slightly from the output.

There should be a better way to do this but this works for now.

**Pull request merged**